calblueprint/ogsc
# Add birth year editing capability via PFEM

* Adds edit icon next to birth year for editing (this doesn't include player name as in the mockup, but @rlhe811 stamped the screenshot and said it will work for now ✔️)
* Group the header-related components into its own component, so it can read from the ProfileContext

## How to Test/Use PR feature

* Login as an admin and validate that the birth year field is editable
* Login as a non-admin and validate that the birth year field edit icon does not appear

## PR Checklist

Does your branch (check if true):


## Screenshots

<img src="https://user-images.githubusercontent.com/2977329/116671316-a75aeb80-a955-11eb-819a-7528b4215df7.png" width="300">

CC: @sydbui
